src/ui/server.js

```diff
@@ -36,6 +36,7 @@ export default function render(
       <meta charset="utf-8">
       <meta name="viewport" content="width=device-width,initial-scale=1" />
       <link rel="shortcut icon" href="${assets.resolve("/favicon.png")}" />
+      <link rel="manifest" href="${assets.resolve("/manifest.json")}" />
```

Comment:
Why is this a part of this PR?

Reply:
I don't know. I have no memory of adding this line, or what it does.